feat: add apprisk rules opt-in injection

aarlaud opened the pull request:
- [x] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
Adds ACCEPT_APPRISK flag loading rule subset based on current broker downstream type, composing the ruleset upon startup.

#### Where should the reviewer start?


#### How should this be manually tested?


#### Any background context you want to provide?


#### Screenshots


#### Additional questions

Commit:
feat: add apprisk rules opt-in injection

ariel-enso commented:
@aarlaud it would be nice to test this branch before merging it, could you provide example of how to use the `ACCEPT_APPRISK` flag? :]

aarlaud commented:
> @aarlaud it would be nice to test this branch before merging it, could you provide example of how to use the `ACCEPT_APPRISK` flag? :]

simply adding the env var ACCEPT_APPRISK=true

ilan-adisa committed:
removed more redundant rules

removed redundant rules

updated github + github-enterprise rules for appRisk

updated github rules for appRisk

added missing endpoint

updated endpoints

added bitbucket endpoints

rollback file changes

added azure devops endpoints

ilan-adisa committed:
update snapshot after updating appRisk rules

Commit:
Merge pull request #665 from snyk/feat/add-apprisk-opt-in-rules-azure-devops

Feat/Added apprisk rules

snyksec commented:
:tada: This PR is included in version 4.171.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/snyk-broker/v/4.171.0)
- [GitHub release](https://github.com/snyk/broker/releases/tag/v4.171.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
